McZapkie/hamulce.cpp

```diff
@@ -118,7 +118,6 @@ double PFVd( double PH, double PL, double const S, double LIM, double const DP )
         LIM = LIM + 1;
         PH = PH + 1.0; // wyzsze cisnienie absolutne
         PL = PL + 1.0; // nizsze cisnienie absolutne
-        assert( PH != PL );
```

Comment:
Just removing this doesn't look correct. div/0 below because `std::sqrt((sg) * (1.0 - sg))`. Probably there should be also protection against sg>1.